PluginGenerator/PluginBuilder.cs

```diff
@@ -17,14 +17,34 @@ public class PluginBuilder
     {
         public const string SONARQUBE_API_VERSION = "4.5.2";
 
+        /// <summary>
+        /// The name of the package in which the plugin will be created
+        /// </summary>
+        private const string CorePluginPackageNameTemplate = "org.sonarqube.plugin.sdk.[PLUGIN_KEY]";
+
+        /// <summary>
+        /// The name of the class that SonarQube should load to discover the
+        /// extensions offered by the plugin
+        /// </summary>
+        private const string CorePluginEntryClassTemplate = "[PLUGIN_PACKGE].Plugin";
```

Comment:
Related to minor clean up to the .java files. There is now a *[PLUGIN_PACKAGE]* token that can be used as a placeholder in other .java files, rather than requiring all .java files to know how the package name is constructed.